<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtImageDemoClass</class>
 <widget class="QMainWindow" name="QtImageDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtImageDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acOpen"/>
   <addaction name="acRun"/>
   <addaction name="acZoom1"/>
   <addaction name="acZoom2"/>
   <addaction name="acZoom3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="acRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="acZoom1">
   <property name="text">
    <string>Zoom1</string>
   </property>
   <property name="toolTip">
    <string>Zoom1</string>
   </property>
  </action>
  <action name="acZoom2">
   <property name="text">
    <string>Zoom2</string>
   </property>
  </action>
  <action name="acZoom3">
   <property name="text">
    <string>Zoom3</string>
   </property>
  </action>
  <action name="acOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtimagedemo.qrc"/>
 </resources>
 <connections/>
</ui>
